Userland/Services/ChessEngine/ChessEngine.h

```diff
@@ -26,8 +33,8 @@ class ChessEngine : public Chess::UCI::Endpoint {
     Function<void(int)> on_quit;
 
 private:
-    ChessEngine(NonnullOwnPtr<Core::File> in, NonnullOwnPtr<Core::File> out)
-        : Endpoint(move(in), move(out))
+    explicit ChessEngine()
```

Comment:
Why is this made `explicit`? I think we only do that for single-argument constructors. We don't really seem to care about initializer lists, for example.

Reply:
The history behind this is that for a short few seconds, `ChessEngine` was indeed a single-arg constructor in my IDE, and then I changed it, and apparently forgot to remove the "explicit". :)